tools/test

```diff
@@ -1,6 +1,5 @@
 #!/bin/sh
 
-ROOT=$(dirname $0)/..
-$ROOT/tools/sigh test
-(cd $ROOT/extension && npm test)
-(cd $ROOT/shell && npm test)
+cd $(dirname $0)/..
```

Comment:
I think we'll need some further work to make this windows compatible. Have you been able to test it? We have a windows machine in syd we (cameron or I) could use tomorrow?

Reply:
I'm willing to revert this change cause I didn't end up using it.